src/forms/textarea.jsx

```diff
@@ -32,7 +32,7 @@ class TextArea extends Input {
         cols={cols}
         disabled={disabled}
         readOnly={readonly}
-        {...omit(other, 'maxHeight', 'dispatch')}
+        {...omit(other, 'maxHeight', 'dispatch', 'fieldName', 'initialHideReveals', 'meta')}
```

Comment:
This feels a little bit wildly arbitrary, and if those props are being passed in by our implementation then we should probably chop them off there.

`maxHeight` and `dispatch` have legit reasons for being included in the component props, and so it's fair to call them out here, but our implementation passing in noise shouldn't be reflected here.